src/Controls/samples/Controls.Sample/Pages/PlatformSpecifics/Windows/WindowsTileBarPage.xaml.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using Maui.Controls.Sample.ViewModels;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.Shapes;
+using Microsoft.Maui.Controls.Xaml;
+
+namespace Maui.Controls.Sample.Pages
+{
+	public partial class WindowsTitleBarPage : ContentPage
+	{
+		TitleBarSampleViewModel _viewModel;
+		TitleBar _customTitleBar;
+
+		public WindowsTitleBarPage()
+		{
+			InitializeComponent();
+
+			_viewModel = new TitleBarSampleViewModel();
+			BindingContext = _viewModel;
+
+			string titleBarXaml =
+				"""
+				<TitleBar
+					xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+					Title="{Binding Title}"
+					Subtitle="{Binding Subtitle}"
+					IsVisible="{Binding ShowTitleBar}"/>
+				""";
+
+			_customTitleBar = new TitleBar().LoadFromXaml(titleBarXaml);
+			_customTitleBar.BindingContext = _viewModel;
+		}
+
+		protected override void OnAppearing()
+		{
+			base.OnAppearing();
+			Window.TitleBar = _customTitleBar;
```

Comment:
Shouldn't this be removed again when the page closes?
It does feel slightly weird to have the contentpage mess with the app-wide window. That might be more appropriate to use AppShell for instead, since that requires to be the root of the window content and hosts pages inside it.